.github/workflows/test-all-packages.yml

```diff
@@ -78,6 +78,38 @@ jobs:
     - name: lint check
       run: yarn lint-check
 
+  benchmark:
+    needs: build
```

Comment:
```suggestion
    needs: build
    # Only run if we were a push to master.
    if: ${{ github.event_name == 'push' }}
```

Reply:
Hmm, why not split it out into its own file and change the conditions by removing `pull_request` below:

```
on:
 push:
   branches: [master]
 pull_request:
```

Reply:
> Hmm, why not split it out into its own file and change the conditions by removing `pull_request` below:

Then it can't depend on the build task, and I'm not sure it can leverage any of the caches from test-all-packages.